### ISSUE TITLE
ReceiverInfo in subscriptions is not well parsed

### ISSUE THREAD

**Issue by jlanza** [ScorpioBroker/ScorpioBroker#405]
Hi,
After the recent update the subscriptions seem not to be well parse. When including ReceiverInfo an invalid entry error is reported.

```
POST http://myscorpio.com/ngsi-ld/v1/subscriptions/
Content-type: application/json

{
 "id": "urn:ngsi-ld:Subscription:subscription-jlanza", 
 "type": "Subscription", 
 "subscriptionName": "Subscription for jlanza", 
 "description": "Notify me on new entity", 
 "entities": [{"type": "MyType"}], 
 "isActive": "true", 
 "notification": {
    "format": "normalized", 
    "endpoint": {
        "uri": "https://myserver.com/ngsi-ld/notifications",
        "accept": "application/ld+json", 
        "receiverInfo": [{"Authorization": "mydata"}]
    }
 }
}
```

An the response is:

```
HTTP/1.1 400 Bad Request
content-length: 171
connection: close
Content-Type: application/json

{
  "type": "https://uri.etsi.org/ngsi-ld/errors/BadRequestData",
  "title": "Bad Request Data.",
  "detail": {
    "message": "The key Authorization is an invalid entry."
  },
  "errorCode": 400
}
```
 
Before it was working without any issue. Actually if I remove the "receiverInfo" from the "endpoint", the subscription is correctly generated.

Thanks for taking care of this issue as the previous ones ;)

**Comment by jlanza** [comment]
Hi,
We have been working a bit more on this issue. It seems that with the former version of ScorpionBroker there was a quite similar error.

POST http://10.10.30.50:9090/ngsi-ld/v1/subscriptions/
Content-type: application/ld+json

{
"id": "urn:ngsi-ld:Subscription:subscription-jlanza-1", 
"type": "Subscription", 
"subscriptionName": "Subscription for jlanza", 
"description": "Notify me on new catalogs", 
"entities": [{"type": "Dataset"}], 
"notification": {
    "endpoint": {
        "uri": "[https://10.10.30.15:5051/ngsi-ld/notifications"](https://10.10.30.15:5051/ngsi-ld/notifications%22),
        "accept": "application/ld+json", 
        "receiverInfo": [{"Authorization": "mydata"}]
    }
},
"@context": "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-v1.6.jsonld"
}

When including "@context" with the NGSI-LD context Scorpio behaves incorrectly, in the same way as the error above. Actually that happens for any aggregated context that includes the NGSI-LD core context.

**Comment by ScorpioBroker** [comment]
was fixed in 4.0.4
